crates/bevy_color/src/color.rs

```diff
@@ -143,7 +143,7 @@ impl Color {
     /// Creates a new [`Color`] object storing a [`Srgba`] color from [`u8`] values.
     ///
     /// A value of 0 is interpreted as 0.0, and a value of 255 is interpreted as 1.0.
-    pub fn srgba_u8(red: u8, green: u8, blue: u8, alpha: u8) -> Self {
+    pub const fn srgba_u8(red: u8, green: u8, blue: u8, alpha: u8) -> Self {
```

Comment:
Won't work because of floating arithmetic.

Reply:
const floating point arithmetic just stabilized :)

Reply:
ohh I see, I was confused by how the author's bevy_color tests could be passing locally.

Reply:
I had forgotten about Bevy's MSRV.
Should I also change the value of `package.rust-version` in Cargo.toml to `1.82.0` with this pull request?

Reply:
Yes, please do.